Dockerfile

```diff
@@ -12,6 +12,7 @@ COPY  jsonrpc2_v2 /analyzer-lsp/jsonrpc2_v2
 COPY lsp /analyzer-lsp/lsp
 COPY parser /analyzer-lsp/parser
 COPY provider /analyzer-lsp/provider
+COPY pkg /analyzer-lsp/pkg
```

Comment:
I think this needs to be updated to be progress

Reply:
Yes.  I just caught that too.  Fixed.